docs/service-catalog/docs/060-osba-overview.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Overview
+type: UI Contracts
+---
+
+UI Contracts are contracts between the Service Catalog views in the Kyma Console UI and the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBA) specification.
+
+There are three types of OSBA fields:
+* mandatory fields, which are crucial to define
+* optional fields, which you can but do not have to define
+* conventions, which are proposed fields that can be changed with your own fallbacks
+
+The Service Catalog is OSBA-compliant, which means that you can register a Service Class that has only the mandatory fields.
```

Comment:
maybe add a sentence
`Nevertheless it is recommended that for better user experience you provide much more detailed class definitions`
?